tensorflow_serving/g3doc/index.md

```diff
@@ -12,7 +12,8 @@ To get started with TensorFlow Serving:
 * Read the [overview](architecture_overview.md)
 * [Set up](setup.md) your environment
 * Do the [basic tutorial](serving_basic.md)
-
+* Or watch the Talk from TensorFlow Dev Summit 2017:
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/q_IkJcPyNl0?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
```

Comment:
This doesn't display in the GitHub README.md: https://github.com/tutmann/serving/blob/patch-1/tensorflow_serving/g3doc/index.md

Could try something like this:
[![Serving Models in Production with TensorFlow Serving](https://img.youtube.com/vi/q_IkJcPyNl0/0.jpg)](http://www.youtube.com/watch?v=q_IkJcPyNl0)